server/controller/chore.js

```diff
@@ -39,13 +39,37 @@ export const createChore = async (req, res) => {
       });
     }
 
+    const lastCheckedOff = []; // this array will contain all time stamps when the chore was last checked off by the user
+
+    // initially the next occurence date will be based on the frequency and interval
+    // mapping interval into days
+    var intervalToDays;
+    if (frequency.interval == "days") {
+        intervalToDays = 1;
+    } else if (frequency.interval == "weeks") {
+        intervalToDays = 7;
+    } else if (frequency.interval == "months") {
+        intervalToDays = 30; // just defaulting to 30
+    } else {
+        intervalToDays = 365; // interval is in years
+    }
```

Comment:
I think it would be good to pull this logic into a separate function. Seems like something we may need elsewhere too!